File: src/main/java/no/nav/sosialhjelp/soknad/consumer/fiks/dto/FilOpplasting.java
package no.nav.sosialhjelp.soknad.consumer.fiks.dto;

import java.io.InputStream;


public class FilOpplasting {

    public FilMetadata metadata;
    public InputStream data;

    public FilOpplasting(FilMetadata metadata, InputStream data) {
        this.metadata = metadata;
        this.data = data;
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/client/fiks/digisosapi/dto/FilOpplasting.kt
package no.nav.sosialhjelp.soknad.client.fiks.digisosapi.dto

import java.io.InputStream

data class FilOpplasting(
    val metadata: FilMetadata,
    val data: InputStream
)

data class FilMetadata(
    val filnavn: String,
    val mimetype: String,
    val storrelse: Long
)
